src/index.css

```diff
@@ -29,6 +29,7 @@ body {
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/199437485-8008902f-c3e0-4aa6-a46d-ffb7bb33b4cc.png)

- [ ] You have added a link to your license file :blush: but you forgot to add a license file in your project. :smile: Kindly add a license file to your project using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: